.github/workflows/ci.yml

```diff
@@ -40,24 +67,14 @@ jobs:
         if: startsWith(matrix.os,'windows')
         run: |
           vcpkg.exe install sqlite3:x64-windows zlib:x64-windows
-          choco upgrade openssl llvm zip psutils strawberryperl -y
+          choco upgrade llvm zip psutils strawberryperl -y
 
       - name: windows env
         if: startsWith(matrix.os,'Windows')
-        uses: allenevans/set-env@57c4e41b5835146bcd9889174001783dd682b6c5
-        with:
-          SQLITE3_LIB_DIR: 'C:\vcpkg\installed\x64-windows\lib'
-          OPENSSL_DIR: 'C:\Program Files\OpenSSL-Win64'
-          LIBCLANG_PATH: 'C:\Program Files\LLVM\bin'
-
-      # this is a hack to fix an issue with building libclang in MSVC
-      # it should be fixed in release 16.9 of MSVC
-      # issue https://github.com/microsoft/STL/issues/1300
-      # temp fix https://github.com/mono/CppSharp/pull/1514/files
-      - name: fix intrin.h file
```

Comment:
Might want to leave this in, just commented out, in the event it is a problem again at some point in future. Remember that quite a bit of time was spent on fixing this when it was a problem.

Reply:
It was fixed upstream

Reply:
Fixed in VS 16.9 - https://github.com/microsoft/STL/issues/1300#issuecomment-789542493 

windows-2019 runners are already on VS 16.10 - https://github.com/actions/virtual-environments/blob/main/images/win/Windows2019-Readme.md#visual-studio-enterprise-2019